packages/components/src/Table/Table.css

```
@@ -1,10 +1,12 @@
 .table {
   --border-color: var(--color-grey--lighter);
   --header-border-color: var(--color-blue);
-  --horizontal-inset: 0.5rem;
-
+  --horizontal-inset: var(--space-base);
   width: 100%;
+
+  box-shadow: 0 0 0 1px var(--border-color);
```

```suggestion
  box-shadow: 0 0 0 var(--border-base) var(--border-color);
```

https://github.com/GetJobber/atlantis/blob/master/packages/design/src/borders.css#L2